lib/view/maalcie/beheer/MaaltijdLijstView.class.php

```diff
@@ -44,7 +44,7 @@ public function getBreadcrumbs() {
 
 	public function view() {
 		$smarty = CsrSmarty::instance();
-		$smarty->assign('stylesheets', ['/dist/css/module-maaltijdlijst.css']);
+		$smarty->assign('stylesheets', ['/dist/css/module/maaltijdlijst.css']);
```

Comment:
uhh. `module-maaltijdlijst.css` klopt, dit is zo gedaan omdat webpack niet goed werkt met output in geneste structuur.

Reply:
Uhm, betekent dat dan dat we handmatig die module moeten hernoemen? Bij mij werkte het namelijk niet. Na deze wijziging werkte het wel

Reply:
Oh, ik denk dat het kwam doordat ik een verkeerde versie van webpack gebruikte.